src/back/spv/block.rs

```diff
@@ -888,6 +888,127 @@ impl<'w> BlockContext<'w> {
                         id,
                         arg0_id,
                     )),
+                    Mf::CountTrailingZeros => {
+                        let uint = crate::ScalarValue::Uint(32);
+                        let int = crate::ScalarValue::Sint(0);
+
+                        let (int_type_id, _int_id) = match *arg_ty {
+                            crate::TypeInner::Vector { size, width, .. } => {
+                                let ty = self.get_type_id(LookupType::Local(LocalType::Value {
+                                    vector_size: Some(size),
+                                    kind: crate::ScalarKind::Sint,
+                                    width,
+                                    pointer_space: None,
+                                }));
+
+                                self.temp_list.clear();
+                                self.temp_list
+                                    .resize(size as _, self.writer.get_constant_scalar(int, width));
+
+                                let id = self.gen_id();
+                                block.body.push(Instruction::constant_composite(
```

Comment:
@teoxoy This isn't necessarily a urgent matter, but is `constant_composite` appropriate in these instances, and if so should be be caching them?

Reply:
I think it's appropriate, not sure how else you'd achieve this result.
We are caching constant scalars, so it would be ideal to cache composites as well.

Reply:
Appropriate in the sense that there's also a `composite_construct` being used to inline vectors. So would these ideally be updated as well or does it only make sense for instances where `get_constant_scalar` is used to compose the vector?

Reply:
Ideally we would put the resulting constant composites together with the other constants.
I see you already opened a PR https://github.com/gfx-rs/naga/pull/2257 👍